src/odemis/gui/win/acquisition.py

```diff
@@ -785,6 +785,10 @@ def __init__(self, parent, orig_tab_data):
         self.btn_secom_acquire.Bind(wx.EVT_BUTTON, self.on_acquire)
         self.Bind(wx.EVT_CLOSE, self.on_close)
 
+        # range for the autofocus (sensible defaults)
```

Comment:
`sensible defaults` still does not tell me that much. Can you update the comment to explain why `(50e-6, 1000e-6)` covers what is needed? You sort of explain it in the description of the PR:

> Previously this was hardcoded at 450um, which meant that for overviews smaller than that only a single autofocus in the centre of the image was done. This was particularly bad on systems with 100x mag

Reply:
Added additional detail about specific values (half a grid square -> half a grid) for range